<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmSetupSerial</class>
 <widget class="QMainWindow" name="FrmSetupSerial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_Info">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>641</width>
      <height>271</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Thông tin chi tiết về thiết bị</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="pushButton_writeSerial">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>220</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Ghi mã thiết bị</string>
     </property>
     <property name="shortcut">
      <string>Return, Enter</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>150</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Mã định danh:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>34</x>
       <y>70</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Phân nhóm:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_groupDevice">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>201</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <item>
      <property name="text">
       <string>Phần mềm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Cảm biến</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mô-đun thực hành</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mô hình thí nghiệm</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="lineEdit_id">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>150</y>
       <width>151</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="maxLength">
      <number>4</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignJustify|Qt::AlignVCenter</set>
     </property>
     <property name="placeholderText">
      <string>gồm 4 kí tự [0-9][A-F]...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>46</x>
       <y>110</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Phân loại:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_typeDevice">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>110</y>
       <width>201</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <item>
      <property name="text">
       <string>Cảm biến nhiệt độ</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Cảm biến độ ẩm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Cảm biến lực</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Cảm biến áp suất</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Cảm biến âm thanh</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>42</x>
       <y>30</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Phiên bản:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_versioneDevice">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>71</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>9</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>20</y>
       <width>81</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Mã thiết bị :</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>180</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>93</y>
       <width>241</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <italic>true</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(240, 240, 240);
color: rgb(50, 50,50);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string>Mã thiết bị có dạng: ABCDEFGH
A          Phiên bản
B          Nhóm ứng dụng
CD       Loại thiết bị
EFGH   Mã định danh thiết bị</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_serialNumber">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>50</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>gồm 8 kí tự [0-9][A-F]...</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_Increase">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>194</y>
       <width>281</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Tăng lên 1 sau khi ghi</string>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>20</y>
       <width>20</width>
       <height>231</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_Cancel">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>286</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(225, 225, 225);</string>
    </property>
    <property name="text">
     <string>Thoát</string>
    </property>
    <property name="shortcut">
     <string>Esc</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>660</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>spinBox_versioneDevice</tabstop>
  <tabstop>comboBox_groupDevice</tabstop>
  <tabstop>comboBox_typeDevice</tabstop>
  <tabstop>lineEdit_id</tabstop>
  <tabstop>lineEdit_serialNumber</tabstop>
  <tabstop>plainTextEdit</tabstop>
  <tabstop>checkBox_Increase</tabstop>
  <tabstop>pushButton_writeSerial</tabstop>
  <tabstop>pushButton_Cancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
